pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -312,9 +312,22 @@ var _ = Describe("Pod Network", func() {
 			for _, chain := range []string{"output", "KUBEVIRT_POSTINBOUND"} {
 				mockNetwork.EXPECT().NftablesAppendRule(proto, "nat", chain, "tcp", "dport", fmt.Sprintf("{ %s }", strings.Join(portsUsedByLiveMigration(), ", ")), GetNFTIPString(proto), "saddr", getLoopbackAdrress(proto), "counter", "return").Return(nil)
 			}
+			srcAddressesToSnat := []string{getLoopbackAdrress(proto)}
+			dstAddressesToDnat := []string{getLoopbackAdrress(proto)}
+			if hasIstioSidecarInjectionEnabled(vm) {
```

Comment:
IMO, `TestPodInterfaceIPBinding` already contains too much, please move this code to the relevant test.

Reply:
[Done](https://github.com/kubevirt/kubevirt/compare/6ad006b36a9513a1eeefbfd5983f45b448dec462..3df75c95cbd5de05b96cc22a350303f8005b5615)